### PR TITLE
CHAT-412 Add Event facility in 1:1 chat rooms

### PR DESCRIPTION
**Pull request by trongtt**

### CODE REVIEW

File: application/src/main/webapp/js/chat.js [hunk 1]
```diff
@@ -203,10 +203,14 @@ var chatApplication = new ChatApplication();
       $toggle.show();
       $(".meeting-action-title").html(ptitle);
       $popup.show();
-      if (toggleClass === "meeting-action-event-panel" && chatApplication.targetUser.indexOf("team-")>-1) {
-        chatApplication.getUsers(chatApplication.targetUser, function (users) {
-          $("#chat-file-target-user").val(users);
-        }, true);
+      if (toggleClass === "meeting-action-event-panel") {
+        if (chatApplication.targetUser.indexOf("team-") > -1) {
```
Comment (thomasdelhomenie) [inline application/src/main/webapp/js/chat.js]: Couldn't we have an issue here with a space containing "team-" in its name ? For "space-team-1".
We should probably test on == 0 instead.

File: application/src/main/webapp/js/chat.js [hunk 2]
```diff
@@ -203,10 +203,14 @@ var chatApplication = new ChatApplication();
...
+          chatApplication.getUsers(chatApplication.targetUser, function (users) { // Team chat room
+            $("#chat-file-target-user").val(users);
+          }, true);
+        } else if (chatApplication.targetUser.indexOf("space-") == -1) { // 1:1 chat room
```
Comment (thomasdelhomenie) [inline application/src/main/webapp/js/chat.js]: Can we have the same issue here with 1:1 room containing "space-" in their name ?

File: application/src/main/webapp/js/chat.js (hunk 1 above)
Comment (trongtt) [inline application/src/main/webapp/js/chat.js]: I think NO.
I see the name pattern here is always "[team- | space-] + [random hash number]" and these kinds of checks are being used everywhere in Chat.

However, I find there is a potential issue with 1:1 chat room that username is being used for targetUser. It is possibly containing "team-" or "space-".
For me, this potential issue could be fixed in other issue. We would introduce something like *chatApplication.typeOfRoom*, and the overall refactoring on these check is needed.

File: application/src/main/webapp/js/chat.js (hunk 2 above)
Comment (trongtt) [inline application/src/main/webapp/js/chat.js]: Yes, same answer as above. This check is being used everywhere in Chat app.

File: application/src/main/webapp/js/chat.js (hunk 1 above)
Comment (thomasdelhomenie) [inline application/src/main/webapp/js/chat.js]: ok, let's open a new issue to improve this and fix this potential issue.

**Comment by thomasdelhomenie** [comment]
LGTM

**Pull request merged**